### ISSUE TITLE
Issue when running ImageTools.align_frames running out of colors to assign for plot

### ISSUE THREAD

**Issue by strampelligiovanni** [kammerje/spaceKLIP#187]
When running ImageTools.align_frames with 10 or more items in the Database the pipeline return the following error:

```
Traceback (most recent call last):
  File "/Users/gstrampelli/miniconda3/envs/spaceklip/lib/python3.11/contextlib.py", line 81, in inner
    return func(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^
  File "/Users/gstrampelli/spaceKLIP/spaceKLIP/imagetools.py", line 2171, in align_frames
    s=5, color=colors[len(seen)], marker=syms[0], 
               ~~~~~~^^^^^^^^^^^
IndexError: list index out of range
```

I believe this is due to the fact that color is defined as `colors = plt.rcParams['axes.prop_cycle'].by_key()['color']` and its length is only 10, so it runs out of colors to assign.

**Comment by wbalmer** [comment]
Bumping this - I ran into this issue when compiling a big library.

**Comment by AarynnCarter** [comment]
Resolved in #189
